src/Util/loader.php

```diff
@@ -33,6 +33,8 @@ function mu_loader($plugins = false, $ps = DIRECTORY_SEPARATOR, $mudir = WPMU_PL
         }
         require_once $mudir . $ps . $plugin;
     }
+
+    do_action('muplugins-loaded');
```

Comment:
There is already a core `do_action( 'muplugins_loaded' );` - if we want something for this library that's fine, but we should probably prefix to avoid confusion/clashes. 

It probably makes sense to use the existing namespace, so something like `do_action( 'lkwdwrd_mupluginloader_muplugins_loaded' )` or something? 

Note it should use underscores as per https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/WordPress/Docs/NamingConventions/ValidHookNameStandard.xml#L5 (I know this library doesn't adhere to these standards too strictly but hopefully eventually it will and it makes sense anyway as we don't have a standard that says otherwise so far)

Reply:
I've updated this as per the recommendations. Thanks!